account_banking_sepa_direct_debit/views/report_sepa_direct_debit_mandate.xml

```diff
@@ -17,41 +18,104 @@
                 p{
                   font-size: 8px;
                 }
+                .native_lang{
+                    font-style: italic;
+                }
+                .native_lang_small{
+                    font-size: 0.8em;
+                }
             </style>
             <div class="row mt0">
                 <div class="col-xs-3">
                     <img t-if="doc.company_id.logo" t-att-src="'data:image/png;base64,%s' % doc.company_id.logo" style="max-height: 45px;"/>
                 </div>
                 <div class="col-xs-12 text-center">
-                    <h4 t-if="doc.scheme != 'B2B'"><strong>Sepa Direct Debit Mandate</strong></h4>
-                    <h4 t-if="doc.scheme == 'B2B'"><strong>Sepa Business-To-Business Direct debit Mandate</strong></h4>
+                    <h4 t-if="doc.scheme != 'B2B'">
+                        <strong>Sepa Direct Debit Mandate</strong>
+                        <t t-if="doc.partner_id.lang not in native_langs">
+                            <br/><span class="native_lang native_lang_small" t-esc="'Sepa Direct Debit Mandate'"/>
+                        </t>
+                    </h4>
+                    <h4 t-if="doc.scheme == 'B2B'">
+                        <strong>Sepa Business-To-Business Direct debit Mandate</strong>
+                        <t t-if="doc.partner_id.lang not in native_langs">
+                            <br/><span class="native_lang native_lang_small" t-esc="'Sepa Business-To-Business Direct debit Mandate'"/>
+                        </t>
+                    </h4>
                 </div>
             </div>
             <div class="row mt8">
                 <div class="col-xs-12">
                     <div class="panel panel-default">
-                        <span class="col-xs-12 text-right" style="font-size:8px;">To be completed by the creditor</span>
+                        <div class="col-xs-12 text-right" style="font-size:8px;">
+                            To be completed by the creditor
```

Comment:
This is not a cosmetic change, I've changed from span to div in order to add inside another span.
Teorically translator dicards spaces before and after string.

Anyway I'm going to check how string is included in .pot and update it in all .po files